### DISCUSSION TITLE
Opening streams from Chatterino

### DISCUSSION THREAD

**Discussion by donnydark0** [Chatterino/chatterino2#3866] [Ideas] [answered]
How it is: Right now you have to right-click, find 'open stream in streamlink/player', click

Fix: Make it quicker

Ideas:
1) Set preferred opening method (streamlink/player/browser)
2) Add a button to open stream
Add option to open stream by double-clicking on tab
Open stream by double-clicking on Streamer Name (instead of opening 'change channel')

**Comment by Mm2PL** [comment, marked as answer]
you can add a hotkey instead
![2022-07-18_17-18](https://user-images.githubusercontent.com/25011746/179544722-4a9823b6-fdff-4aba-9002-b41c8d9df0a5.png)
alternatively
![2022-07-18_17-18_1](https://user-images.githubusercontent.com/25011746/179544765-df267479-f1f1-45b9-9786-9b793029c720.png)

**Reply by donnydark0** [reply]
I was going through the list, but wasn't aware you could add more. Thanks, this works very well. Still an improved GUI method would be nice.
